web/src/settings_users.ts

```diff
@@ -231,10 +251,17 @@ function populate_users(): void {
         failed_listing_users();
     }
 
+    // Populate the table and dropdown for active users
     active_section.create_table(active_user_ids);
+    create_role_filter_dropdown($("#admin-user-list"), active_section, active_user_ids);
+
+    // Populate the table and dropdown for deactivated users
```

Comment:
I think these comments are unnecessary and should be removed; the code structure and function names makes it clear what's happening.